default.nix

```diff
@@ -148,10 +141,10 @@ with rec
                     name = "some-name";
                   }
                   )
-                )];
+                );
             });
   };
 
-{ inherit buildPackageSingleStep buildPackageIncremental crates;
-  buildPackage = buildPackageIncremental;
+{
+  inherit buildPackage;
```

Comment:
I wondering, since `naersk` now exports just this single `buildPackage` function, have you considered just returning `buildPackage` instead of `{ inherit buildPackage; }`?

Reply:
So that'd look like `let pkg = (import naersk {}) src attrs; in ...` ?

Reply:
Yes and you would typically assign the function to a name like:

```
let 
  naerskBuildPackage = import naersk {};
  pkgs = naerskBuildPackage src attrs;
in ...
```

Although that's  not that different than the current:

```
let 
  naersk = import naersk {};
  pkgs = naersk.buildPackage src attrs;
in ...
```

However, if users want that they can always introduce the name spacing themselves like:

```
let 
  naersk = { buildPackage = import naersk {}; };
  pkgs = naersk.buildPackage src attrs;
in ...
```

Oh well, I'll let you decide ;-)

Reply:
Good point. I'll avoid breaking compatibility for now but it'd be good to have a think at some point.